modules/ROOT/pages/dashboards/parameters-and-filters.adoc

```diff
@@ -0,0 +1,115 @@
+= Parameters and filters
+:description: Use parameters in your dashboards to further customize queries or use them in filters for node and relationship properties.
+
+
+== Parameters
+
+Parameters are associated with a specific dashboard.
+Each parameter has a value and type both of which you can set initially. 
+
+Display the parameters of a dashboard by using the parentheses button to show the **Parameters** drawer.
+
+.The parameters drawer
+image::dashboards/parameters-and-filters/parameters-drawer.png[]
+
+The drawer shows the dashboard parameters in alphabetical order.
+Each parameter has an icon next to it hinting at its data type:
+
+* A text icon for strings,
+* A hashtag icon for numbers,
+* A calendar icon for dates,
+* A filter icon for unspecified data types. 
+
+
+=== Creating a parameter
+
+. From the **Parameters** drawer, use the **Add** button.
+. In the dialog, enter a parameter name, select a data type and optionally an initial value for the parameter, then **Save**.
+
+
+=== Editing a parameter
+
+. From the **Parameters** drawer, hover over the parameter you wish to edit.
+. Use the pen button that appears.
+. In the dialog, edit the parameter name, data type and/or initial value, then **Save**.
+
+
+=== Using a parameter in a query
+
+To use a parameter in a query, prefix the parameter name with `$`.
+For example, to use a parameter with a number value of 4 and the name "unit_param", use `$unit_param` in your query:
+
+.Cypher query for a table which displays products with a certain number of units in stock
+[source,cypher]
+----
+MATCH (p:Product)
+WHERE p.unitsInStock = $unit_param
+RETURN p.productName as product, p.unitsInStock AS units
+  LIMIT 10
+----
+
+.Using a parameter in a query to display products with a certain number of units in stock
+image::dashboards/parameters-and-filters/parameter-in-query.png[]
+
+
+=== Deleting a parameter
+
+. From the **Parameters** drawer, hover over the parameter you wish to edit.
```

Comment:
This is also pretty self-explanatory. I'd suggest to rewrite so that it's not a step-by-step.